server/server.js

```diff
@@ -0,0 +1,38 @@
+require('dotenv').config();
+
+const express = require('express');
+
+// express app
+const app = express();
+
+// database
+const connection = require('./database');
+
+// test db connection
+app.get('/', function (req, res) {
+    let sql = "SELECT * FROM initial_broncohacks_db.User"
```

Comment:
Theres prob a better way to test connection but this will work for now, I'll approve it so people can get started